cypress/support/index.d.ts

```diff
@@ -2,7 +2,9 @@
 
 declare namespace Cypress {
   interface Chainable<Subject> {
-    getByDataCy(selector: string): Chainable<any>
+    getByDataQa(selector: string): Chainable<any>
```

Comment:
The method getByDataCy has been renamed to getByDataQa, but the task description mentions to add `data-qa` attributes to elements. Therefore, it might cause misunderstanding. Make sure to name methods according to the attributes you're using in your tests.